python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -178,6 +184,22 @@ def __init__(self, opsimdb, descqa_catalog, dither=True,
             else:
                 raise IOError("Path to Proto DC2 AGN database does not exist.")
 
+        if host_image_dir is None:
```

Comment:
It is possible to instantiate `InstanceCatalogWriter` with `sprinkler=False`, in which case the sprinkler is not run.  In that case, the script should probably not try to include these lens hosts, correct?